### DISCUSSION TITLE
ComfyUI V1, how will this be impacted?

### DISCUSSION THREAD

**Discussion by dreamlogic-X** [TheLastBen/fast-stable-diffusion#2943] [General]
Hey Ben, now that the Comfy devs are moving towards desktop first model for ComfyUI, I'm wondering how it will work to run Comfy on Paperspace in the future? Will it be possible to run V1 through a notebook?

https://blog.comfy.org/comfyui-v1-release/

Thanks

**Comment by TheLastBen** [comment]
ComfyUI windows is just an extra, it will still run on linux, so no worries at all
